### DISCUSSION TITLE
Scenes not coming through to Home

### DISCUSSION THREAD

**Discussion by mhollerb** [milo526/homebridge-tuya-web#226] [Q&A]
Milo, thanks for the plug-in, very cool to add other bulbs to Home. 

I’m having trouble adding Tuya scenes to Home. Scenes is set to true. Should I be seeing the default scenes from the Tuya app (relax, candlelight, focus, party, etc.)? And do you expect them to show as new standalone Home Scenes, or are they options within each bulb’s settings? 

Thanks,
Mat

**Comment by milo526** [comment]
Please post the result of [the steps under "How to check whether the api this library uses can control your device".](https://github.com/milo526/homebridge-tuya-web#how-to-check-whether-the-api-this-library-uses-can-control-your-device)

Without this it is hard for me to help you with regards to this specific issue.
